teachers_digital_platform/jinja2/teachers_digital_platform/prototypes/crt-survey.html

```diff
@@ -1,148 +1,339 @@
+            <div class="o-survey_number">
+                <div class="h3">1.3</div>
+            </div>
+            <div class="o-survey_indicator">
+                <div class="h3">Indicator</div>
+                <p>Investments involve purchase of financial assets to increase wealth.</p>
+            </div>
+            <div class="o-survey_components">
+                <div class="h3">Component</div>
+                <div class="o-survey_component">
+                    <div class="o-survey_question">
+                        <p>The goal of savings is to set aside income for future spending, whereas the goal of investing is to increase wealth over time.</p>
+                    </div>
+                    <div class="o-survey_answer">
+                        <div class="m-form-field
+                                    m-form-field__radio
+                                    m-form-field__lg-target">
+                            <input class="a-radio" id="crt-question_4a" type="radio" name="crt-question-4" value="1">
+                            <label class="a-label" for="crt-question_4a">
+                                Yes
+                            </label>
+                        </div>
+                        <div class="m-form-field
+                                    m-form-field__radio
+                                    m-form-field__lg-target">
+                            <input class="a-radio" id="crt-question_4b" type="radio" name="crt-question-4" value="0">
+                            <label class="a-label" for="crt-question_4b">
+                                No
+                            </label>
+                        </div>
+                    </div>
                 </div>
-                <div class="m-form-field m-form-field__radio">
-                    <input class="a-radio" type="radio" name="radio_3" id="test_6">
-                    <label class="a-label" for="test_6">No</label>
-                </div>
-            </li>
-        </ol>
-        <h3>My notes</h3>
-        <p class="m-form-field m-form-field__textarea">
-            <label class="a-label a-label__heading" for="test_7">
-                Write your notes for this criterion <em>(Optional)</em>
+            </div>
+        </div>
+        <div class="m-form-field m-form-field__textarea">
+            <label class="a-label a-label__heading" for="crt-notes-1">
+                My notes <small class="a-label_helper">(optional)</small>
             </label>
-            <textarea class="a-text-input" id="test_7"></textarea>
-        </p>
+            <p class="u-mb15"><small>Anything you want to note about this criterion? Please do not share any Personally Identifiable Information (PII), including, but not limited to, your name, address, pho number, email address, Social Security number, etc.</small></p>
+            <textarea class="a-text-input
+                            a-text-input__full
+                            u-mw670"
```

Comment:
If we put the `max-width` rule as an enhancement for `m-form-field__textarea` (which I think would be a good idea anyway, to keep line lengths in textareas sane), we can eliminate the need for this utility class.

Reply:
Fixed